src/messages/layouts/MessageLayoutContainer.cpp

```diff
@@ -394,18 +394,18 @@ void MessageLayoutContainer::paintSelection(QPainter &painter, int messageIndex,
                     {
                         returnAfter = true;
                         index = line.startCharIndex;
-                        for (int i = line.startIndex; i < line.endIndex; i++)
```

Comment:
why is this `kk` instead of something descriptive?

Reply:
Happy to rename it, but it did seem confusing to use `i` again inside a loop that is already using `i`.